processors/sync_text_from_documentcloud/processor.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import documentcloud
+import logging
+from .. import base
+logger = logging.getLogger(__name__)
+
+
+def process(conf, conn):
+    query = (
+        'SELECT id, documentcloud_id FROM files'
+        ' WHERE documentcloud_id IS NOT NULL'
```

Comment:
I'd go for a triple quoted multiline string on this one, just for the ease of maintainability. But it really doesn't matter that much in this case, up to you.